apps/web/src/lib/mock/organizations.ts

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable */
+interface OrgMockData {
+  id: string;
+  name: string;
+  ownerId: string;
+}
```

Comment:
Is this `interface` used anywhere else? If not delete it. Add types when it's strictly necessary.